ios/File.swift

```diff
@@ -0,0 +1,9 @@
+//
```

Comment:
what is this file

Reply:
this file needs to make ios animations work.
@n make this the same. please see https://github.com/nicastelo/lottieTest